plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -27,28 +27,40 @@ SecRule TX:nextcloud-rule-exclusions-plugin_enabled "@eq 0" "id:9508099,phase:1,
 # you put something like this in crs-setup.conf:
 #
 # SecRule REQUEST_FILENAME "@rx /(?:remote\.php|index\.php)/" \
-#   "id:9508600,\
-#   phase:2,\
-#   t:none,\
-#   nolog,\
-#   pass,\
-#   setvar:'tx.restricted_extensions=.bak/ .config/ .conf/'"
+#     "id:9508600,\
+#     phase:2,\
+#     t:none,\
+#     nolog,\
+#     pass,\
+#     setvar:'tx.restricted_extensions=.bak/ .config/ .conf/'"
 #
 # Large uploads can be modified with SecRequestBodyLimit. Or they
 # can be more controlled by using the following:
 #
 # SecRule REQUEST_URI "@endsWith /index.php/apps/files/ajax/upload.php" \
-#    "id:9508610,\
-#    phase:1,\
-#    t:none,\
-#    nolog,\
-#    ctl:requestBodyLimit=1073741824"
+#     "id:9508610,\
+#     phase:1,\
+#     t:none,\
+#     nolog,\
+#     ctl:requestBodyLimit=1073741824"
 #
 # Please note, that `ctl:requestBodyLimit` action does not work
 # in libmodsecurity3!
+#
+# The Nextcloud desktop client occasionally sends large request bodies not containing any uploaded files.
+# ModSecurity will block request bodies larger than 131KB, changing the SecRequestBodyNoFilesLimit
+# directive to a higher value for the file path/remote.php/dav/files/ fixes this issue.
+#
+# Nginx libmodsecurity3 Example:
+# location /remote.php/dav/files/ { modsecurity_rules 'SecRequestBodyNoFilesLimit 141662'; }
```

Comment:
Where does this number come from?

Reply:
it came from my own testing, I haven't seen a request body larger than 141kb.

Reply:
Fair enough, just add that to the comment. How did you compute the number?
1024 * 141 = 144384.
144662 / 1024 = 138.341796875.

Reply:
Sorry, I meant bytes not bits, 141KB. I found that number from trial and error.

Reply:
Ok. I suggest rounding up to 144384, then you can just say that 141KB worked for all tested scenarios.